__tests__/app.test.js

```diff
@@ -27,4 +26,41 @@ describe("GET /api/topics", () => {
                 });
             });
     });
+
+    it("GEt /api/articles/:article_id :- should return :200 and array articles with that id", () => {
+        return request(app)
+            .get("/api/articles/3")
+            .expect(200)
+            .then(({ body }) => {
+                const articles = body.articles;
+
+                expect(articles).toHaveLength(1);
+                articles.forEach((articles) => {
+                    expect(articles).toHaveProperty("article_id", expect.any(Number));
```

Comment:
we're not expecting these articles to have any number id we're expecting them to have the id of 3